data-prepper-plugins/aggregate-processor/src/main/java/com/amazon/dataprepper/plugins/processor/aggregate/actions/CombineAggregateAction.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.aggregate.actions;
+
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.event.JacksonEvent;
+import com.amazon.dataprepper.plugins.processor.aggregate.AggregateAction;
+import com.amazon.dataprepper.plugins.processor.aggregate.AggregateActionResponse;
+
+import java.util.Map;
+import java.util.Optional;
+
+public class CombineAggregateAction implements AggregateAction {
+    static final String COMBINED_EVENT_TYPE = "combined_event";
```

Comment:
Should this be a different event type? We may need to get some clarity on when to use different event types.

Reply:
We haven't discussed different event types much, but I think it would be nice to have some ability to tell which Event is the final combined aggregate. I think it makes sense to have here.

Reply:
I am concerned we are going to clutter our event namespace. Let's remove it until we have a necessary use case for it.